README.md

```diff
@@ -281,7 +281,7 @@ Checks if an input element contains a certain value.
 
 ```js
 const myInput = $('input')
-expect(myInput).toHaveValue('us')
```

Comment:
Can we tweak the examples to show both assertions? One for `toHaveValue` and `toHaveValueContaining`?

Reply:
That was already there if I'm reading the README right @christian-bromann.  Example above it, line 265 in READMe.md.  And in the Examples.md both toHaveValue and toHaveValueContaining are present:

```
        expect(myInput).toHaveValueContaining('USER')
        // or pass `containing` as an option
        expect(myInput).toHaveValue('value', 'user', { containing: true, ignoreCase: true })
```

Let me know if that wasn't what you had in mind!

Reply:
Ah, you right. Github didn't show me everything